src/SOS/lldbplugin/services.cpp

```diff
@@ -1342,10 +1345,20 @@ LLDBServices::GetModuleBase(
 
 ULONG64
 LLDBServices::GetModuleSize(
+    ULONG64 baseAddress,
     /* const */ lldb::SBModule& module)
 {
     ULONG64 size = 0;
-
+#if defined(__APPLE__)
+    mach_header_64 header;
+    ULONG read;
+    HRESULT hr = ReadVirtual(baseAddress, &header, sizeof(mach_header_64), &read) == S_OK;
```

Comment:
Instead of piping the address in, could we do `SBModule.GetObjectFileHeaderAddress`?

Reply:
That is the file relative address and not the "loaded" address which needs the SBTarget to compute (see the above GetModuleBase function).